src/Demo.mad

```diff
@@ -0,0 +1,78 @@
+import {} from 'String'
+import {} from 'Number'
+import IO from 'IO'
+
+import {getLocation} from './Browser'
+import {instigate} from './Main'
+
+hex :: String -> String
+hex = (r) => (
+  'hsla(' ++ r ++ ', 100%, 50%, 1)'
```

Comment:
Up to you but you could also use template strings and remove the surrounding parentheses:

```madlib
hex = (r) => `hsla(${r}, 100%, 50%, 1)`
```